cobalt/build/configs/cobalt.gni

```diff
@@ -16,5 +16,5 @@
 # configuration.
 
 declare_args() {
-  is_cobalt = true
+  is_cobalt = false
```

Comment:
This is somewhat surprising since it's in a "cobalt/build/configs/cobalt.gni" folder (two "cobalt"s in the name :)

I guess it's to empty the cobalt/build/configs/chromium_*/args.gn files? 

Since those are "foreigners" in the cobalt folder, it would make sense to me as developer to find a `is_cobalt=false` explicitly there. IOW I think reverting the changes in cobalt/build/configs/chromium_*/args.gn makes sense.

Reply:
I'd maybe just leave a comment as to why this `false` default is put in here. It's just to declare an arg.